build/Jenkinsfile

```diff
@@ -16,31 +16,41 @@ node() {
     }
 
     stage('docker-build') {
-        sh '''
-        docker build -t $docker_server/$docker_repo:$commit_id .
-        '''
-        if (env.BRANCH_NAME == 'staging') {
-            sh '''
-            docker build -t $docker_server/$docker_repo:latest .
-            '''
-        }
+        	sh '''
+        	docker build -t $docker_server/$docker_repo:$commit_id .
+        	'''
+        if (env.BRANCH_NAME == 'main' || env.BRANCH_NAME == 'master') {
+            	sh '''
+           	docker build -t $docker_server/$docker_repo:latest .
+            	'''
+        } else if (env.BRANCH_NAME == 'main' || env.BRANCH_NAME == 'staging'){
```

Comment:
What is the logic here, and why for both main and staging? we are building latest tagged repo ?

Reply:
yes
but of it is for staging then ?
then it will go for 2nd na
?